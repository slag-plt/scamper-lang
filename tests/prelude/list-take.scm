(define l (list 1 2 3 4 5))

(list-take l 0)

(list-take l 3)

(list-take l 5)