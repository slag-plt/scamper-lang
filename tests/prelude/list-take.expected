null
(list 1 2 3)
(list 1 2 3 4 5)
